app/admin/api/index.php

```diff
@@ -53,7 +53,7 @@
 			$a = (array) $a;
 
 			# set class
-			$class = $a['app_permissions']==0||(Config::ValueOf('api_allow_unsafe')!==true && $a['app_security']=="none") ? "alert-danger" : "";
+			$class = $a['app_permissions']==0||(Config::ValueOf('api_allow_unsafe')!==true && $a['app_security']==_("none")) ? "alert-danger" : "";
```

Comment:
Hi,
```php
$a['app_security']==_("none")
```

This shouldn't be translated. We're checking the `app_security` column for a row in the `api` table.
This will be in English and be from options `SET('ssl_code','ssl_token','crypt','user','none')`

``` sql
REATE TABLE `api` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `app_id` varchar(32) NOT NULL DEFAULT '',
  `app_code` varchar(32) NULL DEFAULT '',
  `app_permissions` int(1) DEFAULT '1',
  `app_comment` TEXT  NULL,
  `app_security`SET('ssl_code','ssl_token','crypt','user','none')  NOT NULL  DEFAULT 'ssl_token',

Reply:
O.k., I understand. Will rework this too.